@here/harp-geoutils/lib/coordinates/GeoBox.ts

```diff
@@ -198,6 +198,35 @@ export class GeoBox {
         return new GeoBox(this.southWest, this.northEast);
     }
 
+    /**
+     * Update the bounding box by considering a given point.
+     *
+     * @param point The point that may expand the bounding box.
+     */
+    growToContain(point: GeoCoordinates) {
```

Comment:
I find it weird that we have growToContain in GeoBox, but Box2 in THREE has expandByPoint, I think this will just cause confusion for our users, @ninok do we need to have compatibility with HARP Native?

Reply:
harp-geoutils are not only used in harp.gl, but also e.g. in Hype.
Hype doesn't know anything about three.js and it would be weird for Hype developers to see differences in the web and native foundation libraries that are motivated by three.js.

Reply:
True, but I would argue that the github community using harp.gl is (will be) bigger than the number of hype developers, so we are making it easier to understand for a few but missing the audience.

Reply:
But anyway, @Astrak, merge it anyway, and the community can later decide to rename if necessary, I don't want to block.